lib/rules/no-computed-properties-in-native-classes.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+const utils = require('../utils/utils');
+const ERROR_MESSAGE = `Don't use computed properties with native classes. Use getters or @tracked properties instead.`;
+
+/**
+ * Locates all the ImportDeclaration with computed properties.
+ * @param {Node[]} nodeBody Array of nodes.
+ * @returns {Node[]} Array of nodes with .
+ */
+function findComputedNodes(nodeBody) {
+  const importDeclarations = utils.findNodes(nodeBody, 'ImportDeclaration');
+  return importDeclarations.filter(node => {
+    return (
+      node.source.value === '@ember/object/computed' ||
+      (node.source.value === '@ember/object' &&
+        node.specifiers.find(s => s.local.name === 'computed'))
+    );
+  });
+}
+
+//----------------------------------------------
+// General rule - Do not use computed properties in native classes
+//----------------------------------------------
+
+module.exports = {
+  meta: {
+    docs: {
+      description: '',
+      category: 'Best practices',
+      recommended: true,
+      url:
+        'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/no-computed-properties-in-native-classes.md',
+    },
+    fixable: null, // or "code" or "whitespace"
+  },
+
+  create(context) {
+    let computedNodes = [];
+
+    const report = function(node) {
+      context.report(node, ERROR_MESSAGE);
+    };
+
+    return {
+      Program(node) {
+        computedNodes = findComputedNodes(node.body);
```

Comment:
1. Gather up all the import declarations which reference Computed Properties. 
2. If a `ClassDeclaration` is found in the file, throw a lint error for each computed `ImportDeclaration`